www/install/step_upgrade/step3.php

```diff
@@ -43,55 +43,59 @@
 require_once '../steps/functions.php';
 $template = getTemplate('templates');
 
+require_once realpath(dirname(__FILE__) . "/../../../config/centreon.config.php");
+include_once _CENTREON_PATH_ . "/www/class/centreonDB.class.php";
+
 $title = _('Release notes');
 
-if (is_file('../RELEASENOTES.html')) {
-    $contents = "<div id='releasenotes'></div>";
-} else {
-    $releasenotesContent = '';
-    if (file_exists('../RELEASENOTES')) {
-        $releasenotesContent = file_get_contents('../RELEASENOTES');
+$db = new CentreonDB();
+$res = $db->query("SELECT `value` FROM `informations` WHERE `key` = 'version'");
+$row = $res->fetchRow();
+$current = $row['value'];
+
+$_SESSION['CURRENT_VERSION'] = $current;
+
+/*
+** To find the final version that we should update to, we will look in
+** the www/install/php directory where all PHP update scripts are
+** stored. We will extract the target version from the filename and find
+** the farthest version to the current version.
+*/
+$next = '';
+if ($handle = opendir('../php')) {
+    while (false !== ($file = readdir($handle))) {
+        if (preg_match('/Update-([a-zA-Z0-9\-\.]+)\.php/', $file, $matches)) {
+            if ((version_compare($current, $matches[1]) < 0) &&
+                (empty($next) || (version_compare($next, $matches[1]) < 0))) {
+                $next = $matches[1];
+            }
+        }
     }
-    $contents = "<textarea cols='100' rows='30' readonly>" . $releasenotesContent . "</textarea>";
+    closedir($handle);
 }
 
+$majors = explode('.', $next);
+
+$releaseNoteLink = "https://documentation.centreon.com/" . $majors[0] . '.' . $majors[1] . '/en/releases/centreon-core.md';
+
+$title = _('Release notes');
+
+$contents = '<p><b>' . _('Everything is ready !') . '</b></p>';
+$contents .= '<p>' . _('Your Centreon Platform is about to be upgraded from version') . $current . _(' to ') . $next . '</p>';
+$contents .= '<p>' . _('For further details on changes, please find the complete changelog on ');
+$contents .= '<a href="' . $releaseNoteLink . '"target="_blank" style="text-decoration:underline;font-size:11px">documentation.centreon.com</a></p>';
+
 $template->assign('step', STEP_NUMBER);
 $template->assign('title', $title);
 $template->assign('content', $contents);
 $template->assign('blockPreview', 1);
 $template->display('content.tpl');
 ?>
 <script type='text/javascript'>
-    var step3_s = 10;
+    var step3_s = 3;
```

Comment:
do we still need this timer ?

Reply:
In my opinion no, but I've followed the written story. @cgagnaire maybe we could get rid of this ?

Reply:
Ask @Moujimouja :)

Reply:
I guess we will keep it for now, waiting for the build to be :green_heart: to merge